server/controllers/authController.api.js

```diff
@@ -1,39 +1,84 @@
 'use strict';
 const { User } = require('../models');
-const { encodeToken, mongoDbErrorHandler } = require('../utils');
+const { encodeToken, decodeToken, mongoDbErrorHandler } = require('../utils');
 
 exports.register = (req, res) => {
-  User.create(req.body, (err, user) => {
-    if (err) {
-      mongoDbErrorHandler(err, res);
-    } else {
-      const { name, email } = user;
-      const token = encodeToken({ name, email });
-      return res.status(201).json({ token });
-    }
-  });
-};
+    User.create(req.body, (err, user) => {
+        if (err) {
+            mongoDbErrorHandler(err, res);
+        } else {
+            const { name, email, _id } = user;
+            const token = encodeToken({ name, email, _id });
+            return res.status(201).json({
+                status: 201,
```

Comment:
If we have res.status I do not think we need status in the json response data, as we can read response code on front end